src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: YouTube enhances UX with Media Capabilities API
+subhead: |
+  Ensure buttery smooth playback when your users are on different devices.
+authors:
+  - beaufortfrancois
+date: 2019-06-05
+hero: hero.jpeg
+hero_position: top
+alt: Youtube application screengrab
+description: |
+  In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase
+  in MTBR with only a 0.4% decrease in average resolution of videos served.
+tags:
+  - post
+  - media
+  - ux
+  - case-study
+---
+
+In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase in
+MTBR with only a 0.4% decrease in average resolution of videos served.
+
+{% Aside 'key-term' %}
+MTBR (Mean Time Between Rebuffers) is total play time divided by the number of
+rebuffering events.
+{% endAside %}
+
+## The Problem
+
+Typically, media sites will have several variants that they can present users,
```

Comment:
```suggestion
Typically, media sites have several variants of each video that they can present to users,
```
I'm inferring from the article that the focus here is video, not audio, and I thought it'd be good to make that explicit. Also, it prevents any momentary confusion about what kinds of variants we're talking about.

Reply:
Done.